test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -9072,6 +9072,49 @@ def forward(self, x):
         x = torch.ones(2, 0, 3)
         self.run_test(M(), (x,))
 
+    def test_broat_cast_tensors(self):
+        class M(torch.nn.Module):
```

Comment:
Question: How does broadcast work for scalar tensors, or tensors with a '0' dimension? Can we add a test?

Reply:
For scalar tensor, it is just a tensor with 1 dimension so it works as expected.
For tensor with 0 dimension, it is not supported by broadcast yet, so not necessary to add test.